data/conversations.txt

```diff
@@ -22,22 +22,42 @@ conversation "intro"
 	`	It is all going smoothly until it comes time to fire up the steering engines for a test run... and you realize, much to your embarrassment, that you do not have any steering installed. Time to head back to the outfitter and fix that!`
 	
-conversation "flight check: overheating!"
+conversation "flight check: no steering escort!"
+	scene "scene/engine"
+	`You've heard enough horror stories about the sort of accidents that happen when starship systems malfunction to be completely convinced of the necessity of a pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of your ship warming up, you run through a mental checklist. Computers: check. Life support: check. Engines: check. Steering: check.`
+	`	It is all going smoothly until the captain of the <ship> contacts you and tells you that they have no steering engines. How embarrassing. You'll have to tell the captain of <ship> to remain parked until you arrive to fix it.`
+	
+conversation "flight check: overheating flagship!"
 	scene "scene/engine"
 	`You've heard enough horror stories about the sort of accidents that happen when starship systems malfunction to be completely convinced of the necessity of a pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of the <ship> warming up, you run through a mental checklist. Computers: check. Life support: check.`
 	`	By the time you reach the end of the checklist, it has grown uncomfortably warm. The air smells like ozone and hot metal, and your cameras are showing a plume of steam or smoke escaping from the hull near the engines. If your systems are running this hot while still in the atmosphere, they're likely to overheat the moment you hit hard vacuum. You are going to have to either install cooling systems, or trade out some components for lower-heat alternatives.`
+	
+conversation "flight check: overheating escort!"
+	scene "scene/engine"
+	`You've heard enough horror stories about the sort of accidents that happen when starship systems malfunction to be completely convinced of the necessity of a pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of your ship warming up, you run through a mental checklist. Computers: check. Life support: check. Engines: check. Steering: check.`
+	`	It is all going smoothly until the captain of the <ship> contacts you and tells you that their ship was producing so much heat that they didn't think it would be able to travel anywhere without overheating. How embarrassing. You'll have to tell the captain of the <ship> to remain parked until you arrive to fix it.`
```

Comment:
Nice message, but it doesn't work for ships in the same system as the flagship (since the player already arrived there). Also for some new messages above.